synapse_data_warehouse/synapse/dynamic_tables/V2.28.4__certifiedquizquestion_latest.sql

```diff
@@ -0,0 +1,24 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+CREATE DYNAMIC TABLE IF NOT EXISTS CERTIFIEDQUIZQUESTION_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH latest_unique_rows AS (
+            SELECT
+                *
+            FROM
+                {{database_name}}.synapse_raw.certifiedquizquestionsnapshots --noqa: TMP
+            WHERE
+                SNAPSHOT_TIMESTAMP >= CURRENT_TIMESTAMP - INTERVAL '30 DAYS'
```

Comment:
Do we want to use 30 days here even though the certifiedquizquestionsnapshots does not have change_type=delete?

Reply:
Oops! Thanks for catching it - forgot to push